### PR TITLE
Documentation for time range and drop down parameters

### PR DESCRIPTION
**Pull request by acearun**

### CODE REVIEW

File: Documentation/Parameters/Parameters.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+![Image showing a time range referenced in KQL](../Images/Parameters-Time-InCode.png)
+
+### In Text 
+1. Add a text control to the workbook.
+2. In the markdown, enter `The chosen time range is {TimeRange:label}`
+3. Choose _Done Editing_
+4. The text control will show text: _The chosen time range is Last 24 hours_
+
+## Parameter options
+The _In Text_ section used the `label` of the parameter instead of its value. Parameters expose varuiys such options depending on its type - e.g. time range pickers allows value, label, query, start, end and grain.
```
Comment (ericc1103) [inline Documentation/Parameters/Parameters.md]: Should be `allow`?

File: Documentation/Parameters/Parameters.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Workbook Parameters
+
+Parameters allows workbook authors to collect input from the consumers and reference it in other parts of the workbook – usually to scope the result set or setting the right visual. It is a key capability that allows authors to build interactive reports and experiences.
```
Comment (ericc1103) [inline Documentation/Parameters/Parameters.md]: `allow`?

File: Documentation/Parameters/Parameters.md (hunk 1 above)
Comment (ericc1103) [inline Documentation/Parameters/Parameters.md]: `varuiys`?

File: Documentation/Parameters/Time.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+![Image showing the creation of a time range parameter](../Images/Parameters-Time-Settings.png)
+
+This is how the workbook will look like in read-mode.
+
+![Image showing a time range parameter in read mode](../Images/Parameters-Time.png)
+
+## Referencing a time parameter
+### Via Bindings
+1. Add a query control to the workbook and select an Application Insights resource.
+2. Most workbook controls support a _Time Range_ scope picker. Open the the _Time Range_ drop down and select the `{TimeRange}` in the time rang parameters group at the bottom.
```
Comment (ericc1103) [inline Documentation/Parameters/Time.md]: Please remove additional `the`

File: Documentation/Parameters/Parameters.md (hunk 1 above)
Comment (ericc1103) [inline Documentation/Parameters/Parameters.md]: `The In Text` should be `In the Text`?

Comment (acearun) [reply Documentation/Parameters/Parameters.md]: _In Text_ is a section in the doc.

File: Documentation/Parameters/Parameters.md (hunk 2 above)
Comment (acearun) [reply Documentation/Parameters/Parameters.md]: Made change

File: Documentation/Parameters/Parameters.md (hunk 1 above)
Comment (acearun) [reply Documentation/Parameters/Parameters.md]: Made change

Comment (acearun) [reply Documentation/Parameters/Parameters.md]: Made change

**Pull request merged**